Dockerfile.save

```diff
@@ -0,0 +1,168 @@
+FROM ruby:3.2-slim-bookworm@sha256:6ff55a14560f94d6c199033e4aa90cc7f0b7afaea5a50bc91cfbc4905f366f39 as base
```

Comment:
Does this file need to be checked in? IIUC this is effectively a work file and thus it could be Git ignored?